### PR TITLE
Minor UI update

### PR DESCRIPTION
**Pull request by PedroFonsecaDEV**
## Issue This PR Addresses
No open issue.

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

This is a minor update to our UI:

**1:** `<TelescopeAvatar>`  size changed in `<BannerButtons>`.

**2:** Updated `<TelescopeaAvatar>` to receive blog and link to it (optional). 

**3:** Removed some extra spaces from our UI:
Removed `padding-top` for Desktop and mobile from `postsWrapper` rendered by <Posts>

**4:** Banner Button ScrollDown: Is back to work. So users can use the `scroll-snap` if available (@DukeManh  is working on it) or use the button.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by tonyvugithub** [comment]
@humphd: The end to end tests have failed here as well.

**Comment by birtony** [comment]
@chrispinkney @birtony @DukeManh @tonyvugithub @yuanLeeMidori lets take a look a this PR

File: src/web/src/components/TelescopeAvatar.tsx [hunk 1]
```diff
@@ -20,23 +21,30 @@ const getInitials = (name: string) => {
...
     >
-      {initials}
-    </Avatar>
+      <Avatar
+        alt={name}
+        src={img}
+        style={{
+          width: size,
+          height: size,
+          fontSize: `calc(${size} * 0.43)`,
```
Comment (birtony) [inline src/web/src/components/TelescopeAvatar.tsx:40]: We might want to do this calculation on line 26 and store the result in const, then pass it down

**Review by birtony — APPROVED** [review]

**Review by DukeManh — DISMISSED** [review]

**Comment by PedroFonsecaDEV** [comment]
Rebasing and merging.

**Pull request merged**